.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
 Please consider the following before submitting a pull request:
 
-Guidelines for contributing: https://github.com/chartjs/Chart.js/blob/master/CONTRIBUTING.md
+Guidelines for contributing: http://www.chartjs.org/docs/#notes-contributing
```

Comment:
`http://www.chartjs.org/docs/#notes-contributing` will not be anymore valid once we will deploy the new doc so I would rather link to `https://github.com/chartjs/Chart.js/blob/master/docs/developers/contributing.md`.